woocommerce/woocommerce-android
# Multiple shipping lines support

### Description
This PR releases multiple shipping lines support. It does that by displaying multiple shipping lines in the Order Details screen and the Order Create/Edit screen. Additionally, we updated some analytic events to contain a new property to track the number of shipping lines in an order.

### Testing instructions

TC1 

1. Open an order with shipping lines
2. Check that all sipping lines are displayed in the shipping section

TC2

1. Open the app
2. Tap on Add a new order
3. Add a product to enable the shipping section
4. Tap on add shipping and add a new shipping line
5. Check that the new shipping section is displayed
6. Optionally, you can remove the shipping line just added to check that when there is no shipping line, the shipping section is hidden.
7. Tap on the plus button in the shipping section
8. Add a shipping line line
9. Check that the new shipping line is added to the order
10. Edit one of the shipping lines
11. Check that the changes are persisted
12. Delete one of the shipping lines
13. Check that the shipping line is removed from the order
14. Create the order

TC3

1. Using the order form TC2
2. Tap on Edit to enter the edit flow
3. Tap on the plus button in the shipping section
4. Add a shipping line line
5. Check that the new shipping line is added to the order
6. Edit one of the shipping lines
7. Check that the changes are persisted
8. Delete one of the shipping lines
9. Check that the shipping line is removed from the order

### Images/gif

https://github.com/woocommerce/woocommerce-android/assets/18119390/340672d5-5ab4-4a29-b8f2-42e0c62754c7
